docs/blog/2018-03-07-why-we-created-the-plugin-library/index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Why we created the Gatsby plugin library
+date: 03-07-2018
+author: Shannon Soper
+---
+
+## Why build a plugin / package library
+
+In over 60 interviews with Gatsby users, plugin discoverability appeared as a top pain point (see [blog post on the Gatsby UX research program](/2017-12-20-introducing-the-gatsby-ux-research-program/) for a full list of pain points). So we built a plugin / package library [here](/packages/)!
+
+In this article, I want to explain how UX research drove the development and design of this plugin library and invite you all to follow and contribute as the library continues to improve.
```

Comment:
need to use "we" consistently through the article

Reply:
Ok. I got feedback from others on my first article "Introducing the Gatsby UX research program" that I ought to use "I" throughout the article. I don't really care which one, as long as it's consistent haha. So should I switch to "we" now?

Reply:
Haha. Looking at this again, I think this use of "I" is fine in this one spot, since you're referring to yourself as the author of the piece, though I'd argue that there's probably a more powerful way of writing above sentence, eg:

"This article will be talking through our thought processes as we designed & developed this plugin library. We'd love your help contributing as the library continues to improve!"

When you're talking about the user research, using we is better, eg:

We compiled what’s called an empathy map....
We wanted to know what keywords people might use....